libraries/dawg-show/src/main/webapp/views/powerPrompt.jsp

```diff
@@ -29,6 +30,7 @@ Prompt for giving different options concerning power to an stb
         <font size="5px" color="#dedbdb"><b>What do you want to do?</b></font>
     </div>
     <div style="padding:30px;background-color:#dedbdb">
+        <input style='font-size:17px; display: <%= (Boolean) request.getAttribute("isXR11") ? "block;" : "none;" %>' type="button" value="Send Power Key" onclick="sendPowerKey()" />
```

Comment:
talked to product, we only want IR power.  We don't want the ability to power cycle using the power controller